sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/ITimeSeriesId.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Define Time Series Id specific properties and methods to be implemented by the different implementations of Time Series Id classes.
+    /// </summary>
+    public interface ITimeSeriesId
```

Comment:
What do you think about having a `GetId` method or `Id` property that returns the Id as a composite string?

Reply:
Thought about this one. I like the idea. So for now, since the keys of an Id can be of type string, we can do 
`string.Join(",", this.ToArray())`

But how do we handle this once TSI supports primitive types?

Reply:
Rely on object.ToString()?

I can think of several ways to approach this, but just to get the idea across, here's one way.

```csharp
if (_thirdKey.IsUsed)
{
    return $"{_firstKey}{_secondKey}{_thirdKey}";
}

if (_secondKey.IsUsed)
{
    return $"{_firstKey}{_secondKey};
}

return _firstKey.ToString();
```